tutorials/inputs/input_examples.rst

```diff
@@ -297,12 +297,12 @@ also counts as a button - two buttons, to be precise, with both
     {
         if (inputEvent is InputEventMouseButton mouseEvent && mouseEvent.Pressed)
         {
-            switch ((ButtonList)mouseEvent.ButtonIndex)
+            switch (mouseEvent.ButtonIndex)
             {
-                case ButtonList.Left:
+                case MouseButton.Left:
                     GD.Print("Left button was clicked at ", {mouseEvent.Position});
```

Comment:
While you're updating this example, it would be great to fix this print as well.

```suggestion
                    GD.Print("Left button was clicked at ", mouseEvent.Position);
```